### PR TITLE
Add switches for Quiet, Turbo and Eco mode

### PR DESCRIPTION
**Pull request by macjl**
Please review my code, as I'm not a developper !!

### CODE REVIEW

File: config.schema.json [hunk 1]
```diff
@@ -89,6 +89,21 @@
         "title": "Light",
         "type": "boolean",
         "default": true
+      },
+      "turboswitch": {
+        "title": "Turbo Switch",
```
Comment (jhthorsen) [inline config.schema.json:94]: I wonder if this should be changed to "Enable Turbo Switch Homekit service", to make it more descriptive. Same goes for the one below. (Open for other suggestions)

**Review by jhthorsen — CHANGES_REQUESTED** [review]
Thanks for the contribution!

I think most of the changes makes sense, but I do wonder if there's a bug where I mentioned the indentation. A test in https://github.com/jhthorsen/homebridge-tasmota-aircon-http/blob/main/test/plugin.js could prove that... I can write the test for you, but I need to know the intention first.

Further more, all the nitpicking from my side is simply because "git blame" will last longer if the code is sorted properly. It also makes i a lot easier for me to maintain it later on when thinks are in a predictable order.

**Comment by macjl** [comment]
Hello,

Thank you for your feedbacks. I cleaned the code. Tell me if it's OK for you.

Regards,

**Comment by macjl** [comment]
I avec added a modification, to ensure that the switches aren't set to Off, when an automation put a switch to Off the is already in that state.

**Comment by macjl** [comment]
Hello,
Do you think everything is clean now? Or did I miss something ?

**Review by jhthorsen — APPROVED** [review]
Looks good. There isn't any cuddled elses in the code, but I won't hold back the PR because of that.
There's also no tests, so I hope it won't break in the future. I'll try to write some once I find some extra time, unless you beat me to it.

**Comment by jhthorsen** [comment]
I would like to mention you as a author/contributor. If that is ok, then which name do you want me to use? Git log has your email, but I'm not sure how to translate that into your name.

**Comment by macjl** [comment]
Thank you
I will check the tests this week-end. I don’t know how it works, that’s why I didn’t make anything on this before.
For mentioning my contribution, you can use my real name : Jean-Laurent Girod 

I also have some proposition for other improvements. I will open others issue for that.

**Pull request merged**